.github/workflows/package-smoke-test.yaml

```
@@ -166,6 +166,16 @@ jobs:
          - name: Install Playwright browsers
            run: bunx playwright install --with-deps chromium
 
+         - name: Clean nested dependencies in apps
+           run: |
```

actionlint (1.7.9)

shellcheck reported issue in this script: SC2162:info:3:73: read without -r will mangle backslashes

(shellcheck)